### PR TITLE
gameManagment: Added various defination's to all the functions

### PR DESCRIPTION
**Pull request by abhishek20123g**
This commit adds definition to all the skeleton
which were previously defined.

Note(gameMangment): None

### CODE REVIEW

File: gameManagment/enums/cardConstants.py [hunk 1]
```diff
@@ -5,10 +5,46 @@
...
+TEN     = 10
+JACK    = 11
+QUEEN   = 12
+KING    = 13
+ACE     = 14
+
+# List of all valid ranks.
+VALID_RANKS = [THREE, FOUR, FIVE, SIX, SEVEN, EIGHT, NINE, TEN, JACK, QUEEN, KING, ACE]
+
+cardScore = {
```
Comment (keshu6598) [inline gameManagment/enums/cardConstants.py:38]: define Two also

Comment (keshu6598) [reply gameManagment/enums/cardConstants.py:38]: we can use the same library for other games also

File: gameManagment/cardLogic/Card.py [hunk 2]
```diff
@@ -2,34 +2,52 @@
...
     Generate list of 52 cards in random order.
     """
-    pass
+    all52Cards = []
+    for card_value in cardConstants.VALID_RANKS:
+        for card_suit in cardConstants.VALID_SUITS:
+            all52Cards.append(Card(card_value, card_suit))
+
+    random.shuffle(all52Cards)
+    random.shuffle(all52Cards)
```
Comment (keshu6598) [inline gameManagment/cardLogic/Card.py:53]: why 2 times shuffle

Comment (abhishek20123g) [reply gameManagment/cardLogic/Card.py:53]: Extra precuatuion

File: gameManagment/enums/cardConstants.py (hunk 1 above)
Comment (abhishek20123g) [reply gameManagment/enums/cardConstants.py:38]: k

**Pull request closed without merging**